### ISSUE TITLE
New Feature: Generate autoload script

### ISSUE THREAD

**Issue by TheNoobDucky** [TheNoobDucky/Daz-Package-Manager#14]
Feature description: Add a way to automatically create a script that will add the associated virtual folder as base directory and then load the scene.

Implementation notes:
Add a button to write out the following to a Load_<scenename>.dsa file. Replace {virtual folder} and {scene file} with correct location.

```
(function() {
	var virtualFolder = "**{Virtual folder}**";
	var sceneFile = "**{scene file}**";
	var contentManager = App.getContentMgr();
	//contentManager.removeAllContentDirectories(); // Uncomment if currently added base directories are not wanted.
	contentManager.addContentDirectory(virtualFolder); // Include the folder containing asset needed for the scene.
	contentManager.openFile(sceneFile,false); // Load the scene.
})(); 
```
Replace **virtual folder** with base directory you want to be added and **scene file** with your file location.‘

Note: only work with ascii paths

**Comment by TheNoobDucky** [comment]
Implemented in private build.

**Comment by TheNoobDucky** [comment]
Resolved by #18
